SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -452,6 +452,7 @@ UITable.propTypes = {
   keyGetter: PropTypes.func.isRequired,
   children: PropTypes.arrayOf(PropTypes.node).isRequired,
   paginated: PropTypes.bool,
+  isTruelyInfinite: PropTypes.bool, // Will render all rows even if there are 1,000,000
```

Comment:
can we think of a better name for this prop type? we shouldn't be depending on comments to explain what a prop does. furthermore, why cant `paginated` set to `false` support our use case here?